src/concepts/html/components.md

```diff
@@ -63,6 +63,8 @@ impl Component for Container {
 ```
 {% endcode %}
 
+{% hint style="info" %} Types for which you derive Properties must also implement Clone. This can be done by either using #[derive(Properties, Clone) or manually implementing Clone for your type. {% endhint %}
```

Comment:
```suggestion
{% hint style="info" %} Types for which you derive `Properties` must also implement `Clone`. This can be done by either using `#[derive(Properties, Clone)]` or manually implementing Clone for your type. {% endhint %}
```